Description: use newer major version of crate which
Author: Blair Noctis <ncts@debian.org>
Bug-Debian: https://bugs.debian.org/1110142
Forwarded: not-needed
Last-Update: 2025-08-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -101,7 +101,7 @@
 uuid = { version = "1.9", features = ["v4"] }
 walkdir = "2"
 # by default which pulls in an outdated failure version
-which = { version = "6", default-features = false }
+which = { version = "8", features = ["real-sys"] }
 zip = { version = "0.6", default-features = false }
 zstd = "0.13"
 
